### DISCUSSION TITLE
I am getting NoMethodError in Avo::Associations#create for has_many Polymorphic associations when Attaching to a model

### DISCUSSION THREAD

**Discussion by djalmaaraujo** [avo-hq/avo#2052] [Help]
I have this Comment resource:

```
class Avo::Resources::Comment < Avo::BaseResource
  def fields
    field :id, as: :id
    field :name, as: :text do
      record.user.name
    end
    field :content, as: :textarea
    field :user, as: :belongs_to
    field :commentable, as: :has_many, polymorphic_as: :commentable, types: [::Meeting, ::User], polymorphic_help: "Choose the type of record to review", help: "Choose the record you need."
  end
end

```

The `Comment` is a polymorphic model in an existing app. I am trying to attach a comment to a meeting (already works, AVO is new in this app), but I get this error after selecting the comment in the modal:

Step 0:

<img width="1414" alt="image" src="https://github.com/avo-hq/avo/assets/3402/9c862e39-56be-4948-9e2b-f78cde6c1074">

Step 1: 

<img width="1039" alt="image" src="https://github.com/avo-hq/avo/assets/3402/0a8b285c-ac68-4d0b-bed0-c92fbd6564f2">

Step 2:

After pressing Attach, this is the error I get:

<img width="1007" alt="image" src="https://github.com/avo-hq/avo/assets/3402/d6492a8a-eae9-4421-a78a-838e7dfed18c">

Meeting resource:

```
class Avo::Resources::Meeting < Avo::BaseResource
  self.includes = [:owner]

  def fields
    field :id, as: :id
    field :status, as: :select, enum: ::Meeting.statuses
    field :start_at, as: :date_time
    field :end_at, as: :date_time
    field :pid, as: :text
    field :location_type, as: :select, enum: ::Meeting.location_types
    field :owner_id, as: :number
    field :cancel_reason, as: :text
    field :comments, as: :has_many
    field :tag_options, as: :has_many, through: :tags
    field :owner, as: :belongs_to
    field :meeting_participants, as: :has_many
  end
end
```

Do you have any ideas about what it could be?

**Comment by adrianthedev** [comment]
Is this fixed @djalmaaraujo?
